assets/js/script.js

```diff
@@ -0,0 +1,190 @@
+
+
+
+  function restartGame(){
+    $(".landingScreen").hide()
+    setInterval(startMob, 1000)
+    setInterval(bulletCollisionCheck, 10)
+    setInterval(playerMobCollisionCheck,300)
+    playerHealth = 100
+    mobHealth = 50
+    $mob.css("left","205px")
+    gameEnd = false
+  }
+
+
+  function checkHeight(){ // check player height and drop if he is floating
+    var $player = $('.player')
+    var $playPos = $player.position()
+      if ($playPos.top !== 560) {$player.css('top', `560px`)}
+  }
+
+  $body.on('keydown', function (e) {
+    var $player = $('.player')
+    var $playPos = $player.position()
+    keys[e.keyCode] = true
+    $player.css("webkitAnimationPlayState","running")
+
+    if (keys[32] && keys[39] && $playPos.top === 560 && $playPos.left < 1240){ //right and up
+    $player.css('top', `${$playPos.top -= 200}px`)
+    $player.css('left', `${$playPos.left += 200}px`)
+    $player.css("transform","scaleX(-1)")
+    fireDirection = "right"
+    }
+    if(keys[37] && keys[32] && $playPos.top === 560 && $playPos.left >201){  //left and up
+    $player.css('top', `${$playPos.top -= 200}px`)
+    $player.css('left', `${$playPos.left -= 200}px`)
+    $player.css("transform","scaleX(1)")
+    fireDirection = "left"
+    }
+    if (keys[37] && $playPos.left >151) {
+    $player.css('left', `${$playPos.left -= 150}px`)
+    $player.css("transform","scaleX(1)")
+    fireDirection = "left"
+  } //left
+    if (keys[32] && $playPos.top === 560) {$player.css('top', `${$playPos.top -= 300}px`)}   //up
+    if (keys[39] && $playPos.left < 1480) {
+    $player.css('left', `${$playPos.left += 150}px`)
+    $player.css("transform","scaleX(-1)")
+    fireDirection = "right"} //right
+    if (keys[67]) {fireBullet()}
+    e.preventDefault() // prevent the default action (scroll / move caret)
+  })
+
+  $body.on('keyup', function (e) {
+    keys[e.which] = false
+    var $player = $('.player')
+    var $playPos = $player.position()
+    $player.css("webkitAnimationPlayState","paused")
+    // console.log(` current position ${$playPos.top}`)
+    if ($playPos.top !== 560) { $player.css('top', `560px`) }
+  })
+
+  function startMob () {
+    $mobPosition = $mob.position()
+    $playPos = $player.position()
+    mobPosLeft = $mobPosition.left
+    if (mobPosLeft === 205) {
+      $mob.css('left', "1490px")
+      $mob.removeClass("mob")
+      $mob.addClass("mobRight")
+    } else if (mobPosLeft === 1490) {
+      $mob.css('left', "205px")
+      $mob.removeClass("mobRight")
+      $mob.addClass("mob")
+    }
+
+  }
+
+
+
+
+  function fireBullet () {
+    $playPos = $player.position()
+    $mobPos = $mob.position()
+    $bullet.show()
+    $bulletLoc = $bullet.position()
+    $bulletExact = $bulletLoc.left
+    $bulletTop = $playPos.top
+    $bulletLeft= $playPos.left
+    $bullet.css('left',$playPos.left)
+    if(fireDirection === "right"){
+      $bullet.css('left', `${$bulletExact + (1600-$bulletExact)}px`)
```

Comment:
why is $bulletExact use here? why not $playerPos.left? to make it start at player position?